sdk/extensions/Azure.Extensions.AspNetCore.DataProtection.Keys/src/AzureDataProtectionKeyVaultKeyBuilderExtensions.cs

```diff
@@ -54,5 +55,58 @@ public static IDataProtectionBuilder ProtectKeysWithAzureKeyVault(this IDataProt
 
             return builder;
         }
+
+        /// <summary>
+        /// Configures the data protection system to protect keys with specified key in Azure KeyVault.
+        /// </summary>
+        /// <param name="builder">The builder instance to modify.</param>
+        /// <param name="keyResolverFactory">The factory delegate to create the <see cref="IKeyEncryptionKeyResolver"/> to use for Key Vault access.</param>
```

Comment:
Looks like the param doc ordering got swapped.

Reply:
That was me copying and pasting from the existing method. I've reordered that one too!